contracts/eden/src/atomicassets.cpp

```diff
@@ -51,4 +59,67 @@ namespace eden::atomicassets
       return result;
    }
 
+   void init_collection(eosio::name contract,
+                        eosio::name self,
+                        eosio::name collection,
+                        eosio::name schema_name)
+   {
+      ::atomicassets::collections_t collections(contract, contract.value);
+      if (auto pos = collections.find(collection.value); pos != collections.end())
+      {
+         eosio::check(pos->allow_notify, "Notifications are required");
+         eosio::check(std::find(pos->authorized_accounts.begin(), pos->authorized_accounts.end(),
+                                self) != pos->authorized_accounts.end(),
+                      "Contract is not authorized for the collection");
+         if (std::find(pos->notify_accounts.begin(), pos->notify_accounts.end(), self) ==
+             pos->notify_accounts.end())
+         {
+            // This will fail if the contract is not the author of the collection.
+            // In that case, it's the responsibility of the author to make sure
+            // that notifications are set up.
+            actions::addnotifyacc(contract, {self, "active"_n}).send(collection, self);
+         }
+      }
+      else
+      {
+         actions::createcol(contract, {self, "active"_n})
+             .send(self, collection, true, std::vector{self}, std::vector{self}, 0.05,
+                   atomicassets::attribute_map{});
+      }
+
+      std::vector<format> schema{{"edenacc", "string"}, {"name", "string"},
+                                 {"img", "string"},     {"bio", "string"},
+                                 {"social", "string"},  {"inductionvid", "string"}};
```

Comment:
Can we change from `edenacc` to `account` across the board? (to support multiple communities)

@brandonfancher I think that according to the best practices meeting we need these two changes:
1. rename `inductionvid` to `video`
2. new copyrights (or attributions) field? to follow the suggestion to credit the author of the picture art?

Reply:
Agreed. I'll create an issue for this. Thanks!